src/contracts/middleware/BLSIndexRegistryCoordinator.sol

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "../interfaces/IRegistryCoordinator.sol";
+import "../interfaces/IBLSPubkeyRegistry.sol";
+import "../interfaces/IIndexRegistry.sol";
+
+import "../libraries/BytesArrayBitmaps.sol";
+
+import "./StakeRegistry.sol";
+
```

Comment:
this should have like a paragraph at the top explaining what this contract is supposed to be/do.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/b6abc46f2986de3de4965a8a4ad95d0baa573fb7